openreview/venue_request/process/revisionProcess.py

```diff
@@ -132,6 +190,14 @@ def process(client, note, invitation):
                     submission_readers=[openreview.SubmissionStage.Readers.SENIOR_AREA_CHAIRS_ASSIGNED, openreview.SubmissionStage.Readers.AREA_CHAIRS_ASSIGNED, openreview.SubmissionStage.Readers.REVIEWERS_ASSIGNED]
 
             conference.post_decision_stage(reveal_all_authors,reveal_authors_accepted,decision_heading_map=forum_note.content.get('home_page_tab_names'), submission_readers=submission_readers)
+            if forum_note.content['send_decision_notifications'] == 'Yes, send an email notification to the authors':
+                decision_options = forum_note.content.get('decision_options')
```

Comment:
if the PCs left this field empty in the decision stage (to use the default values), then we get an error here when we try to get the decision options and `strip()`

Reply:
Fixed.